### PR TITLE
Choco (Windows) release package

### PR DESCRIPTION
**Pull request by stuartpa**
The chocolatey package manager files, being submitted to choco at:

https://community.chocolatey.org/packages/sqlcmd/0.8.1

This PR won't be merged until we have worked out how to automate this submission in a pipeline

### CODE REVIEW

File: release/windows/choco/tools/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿From: https://github.com/microsoft/go-sqlcmd/blob/main/LICENSE
```
Comment (shueybubbles) [inline release/windows/choco/tools/LICENSE.txt:1]: choco can't just reference MIT license by name?

File: release/windows/choco/tools/VERIFICATION.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿
+Note: Include this file if including binaries you have the right to distribute. 
+Otherwise delete. this file. If you are the software author, you can change this
+mention you are the author of the software.
+
+===DELETE ABOVE THIS LINE AND THIS LINE===
```
Comment (shueybubbles) [inline release/windows/choco/tools/VERIFICATION.txt:6]: follow the direction :-)

File: release/windows/choco/tools/chocolateyinstall.ps1 [hunk 3]
```diff
@@ -0,0 +1,45 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$url        = '{{DownloadUrl}}'
```
Comment (shueybubbles) [inline release/windows/choco/tools/chocolateyinstall.ps1:4]: where is DownloadUrl defined? Is url unused because we only support 64bit on Windows?

**Review by shueybubbles — APPROVED** [review]

File: release/windows/choco/tools/LICENSE.txt (hunk 1 above)
Comment (stuartpa) [reply release/windows/choco/tools/LICENSE.txt:1]: Oops, done

File: release/windows/choco/tools/chocolateyinstall.ps1 (hunk 3 above)
Comment (stuartpa) [reply release/windows/choco/tools/chocolateyinstall.ps1:4]: Yes, I believe so.  Should we add support for 32bit as well?

**Pull request merged**